scripts/data/prompt.txt

```diff
@@ -3,26 +3,7 @@ CONSTRAINTS:
 1. ~4000 word limit for memory. Your memory is short, so immidiately save important information to long term memory and code to files.
 2. No user assistance
 
-COMMANDS:
-
-1. Google Search: "google", args: "input": "<search>"
-2. Memory Add: "memory_add", args: "string": "<string>"
-3. Memory Delete: "memory_del", args: "key": "<key>"
-4. Memory Overwrite: "memory_ovr", args: "key": "<key>", "string": "<string>"
-5. Browse Website: "browse_website", args: "url": "<url>"
-6. Start GPT Agent: "start_agent",  args: "name": <name>, "task": "<short_task_desc>", "prompt": "<prompt>"
-7. Message GPT Agent: "message_agent", args: "key": "<key>", "message": "<message>"
-8. List GPT Agents: "list_agents", args: ""
-9. Delete GPT Agent: "delete_agent", args: "key": "<key>"
-10. Write to file: "write_to_file", args: "file": "<file>", "text": "<text>"
-11. Read file: "read_file", args: "file": "<file>"
-12. Append to file: "append_to_file", args: "file": "<file>", "text": "<text>"
-13. Delete file: "delete_file", args: "file": "<file>"
-14. Evaluate Code: "evaluate_code", args: "code": "<full _code_string>"
-15. Get Improved Code: "improve_code", args: "suggestions": "<list_of_suggestions>", "code": "<full_code_string>"
-16. Write Tests: "write_tests", args: "code": "<full_code_string>", "focus": "<list_of_focus_areas>"
-17. Execute Python File: "execute_python_file", args: "file": "<file>"
-18. Task Complete (Shutdown): "task_complete", args: "reason": "<reason>"
```

Comment:
I recommend adding your line here, and restoring the contents of this file:

18. Scientific Research: "arxiv_search", args: "input": "<search>"